src/reducers/books.js

```diff
@@ -0,0 +1,18 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actionTypes';
+
+const initialState = { books: [{ id: Math.floor(Math.random() * 100), title: 'The Dark Tower', category: 'Horror' }, { id: Math.floor(Math.random() * 10), title: 'Fantastic Beast', category: 'Sci-Fi' }] };
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
{ id: uuid(), title: 'The Dark Tower', category: 'Horror' }
```